android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt

```diff
@@ -48,10 +48,10 @@ object Utils {
         }
 
         @JvmStatic
-        fun checkStringEmpty(string: String): String {
+        fun checkStringEmpty(string: String?): String {
             var finalString = ""
             if (!isEmpty(string)) {
-                finalString = string
+                finalString = string!!
```

Comment:
It's better to just replace this call with `throw new NullPointerException()`

Reply:
Okay, I will update the PR